db/scripts/seed_license_users.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# heroku run:detached --tail \
+#   rails runner db/scripts/seed_license_users.rb
+
+licenses = License.where_assoc_not_exists(:license_users)
+                  .where_assoc_exists(:user)
+                  .includes(:user)
+
+Rails.logger.info "Seeding #{licenses.count} license_users"
+
+licenses.find_each do |license|
+  Rails.logger.info "Seeding license_user for license=#{license.id} <> user=#{license.user_id}"
+
+  license.users << license.user
```

Comment:
Should probably add a sleep in here since this is going to be a large migration.